www/styles/Shared.module.css

```diff
@@ -1,7 +1,6 @@
 .card {
   color: var(--agendaText);
-  background-color: var(--backgroundTwo);
-  border: 1px solid var(--agendaSecondary);
+  border: 1px solid var(--agenda);
```

Comment:
Why did you name the colors this way?

Reply:
@zbarnz legacy systems support

Reply:
@thudsonbu

Reply:
@thudsonbu 
@zbarnz 
@thudsonbu :brain"